Homework_8/Homework8.html

```diff
@@ -0,0 +1,180 @@
+    name : "Women's Dress Sleeveless",
+    description : " Sleeveless Cocktail Dresses till Knee Length",
+    Price: "37€" ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://images-na.ssl-images-amazon.com/images/I/81Yq1G6O-CL._AC_UY445_.jpg" ,
+    name : "Women's Maxi Floral dress",
+    description : " Multicolored casual Floral dress with line",
+    Price: "25€"  ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRF1avcP5Nli-Mu5mYvk1HvIhqu8QpFFumFonCHNNBwmuP9_1f4Qox_dJLTiM9ep58ZFTXEcHE&usqp=CAc",
+    name : "Women's Summer dress",
+    description : " Long Casual Elegant Dresses Party Dress ",
+    Price: "35€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRMancKgpx0vgr9eGIYERMONz6vWT-F5XuMgWmBnrjbOD04KCmQ0KYi-lEVsTrPe6hb264ZLrhU&usqp=CAc",
+    name : "Women's Nightie",
+    description : " Short,pleasantly loose cut ,100% cotton",
+    Price: "15€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+}
+
+] ;
+
+
+    
+
+
+    dressListCmp.innerHTML = `
```

Comment:
when im using for loop.im not able to access the add to cart button.How can i access the add to cart button in order to push the elements from the array.

Reply:
you need to create button inside array with document.createElement
and then do something like this:
```javascript
// all this should happen inside your loop in dressListInput function
cosnt addToCartBtn = document.createElement('button');
addToCartBtn.addEventListener('click', function() {
  showCart.push(dressList[i]);
});
```

this would callback trick would give you access to element to which this Add button relates to.